dev/com.ibm.ws.ejbcontainer_test/test/com/ibm/ws/ejbcontainer/osgi/internal/AnnoMockery.java

```diff
@@ -131,13 +132,20 @@ private void mockInfoAnnotations(final Info info, final AnnotatedElement element
                 for (final Annotation ann : element.getAnnotations()) {
                     final AnnotationInfo annInfo = mockAnnotationInfo(ann);
                     Class<? extends Annotation> annType = ann.annotationType();
-                    allowing(info).getAnnotation(annType);
+                    // Class name is used for ManagedBean rather than Class since the Class may not exist
+                    if (annType_ManagedBean.equals(annType.getName())) {
+                        allowing(info).getAnnotation(annType_ManagedBean);
```

Comment:
In the test it will exist so you can simplify this and get rid of the static String

```suggestion
                    if (ManagedBean.class.equals(annType)) {
                        allowing(info).getAnnotation(ManagedBean.class.getName());
```

Reply:
Changed as requested.